src/Product.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+namespace Netzstrategen\MultiStock;
+
+/**
+ * WooCommerce Product related functionalities.
+ */
+class Product {
+
+  /**
+   * Adds custom fields to WooCommerce simple products.
+   *
+   * @implements woocommerce_product_options_inventory_product_data
+   */
+  public static function woocommerce_product_options_inventory_product_data() {
+    if ($warehouses = Config::get('warehouses')) {
+      static::renderSimpleProductsCustomFields($warehouses, 'Warehouses', 'warehouse');
+    }
+
+    if ($showrooms = Config::get('showrooms')) {
+      static::renderSimpleProductsCustomFields($showrooms, 'Showrooms', 'showroom');
+    }
+  }
+
+  /**
+   * Renders simple product custom fields.
+   *
+   * @param array $fields
+   *   Fields to render.
+   * @param string $sectionTitle
+   *   Title of the fields section.
+   * @param string $fieldsIdPrefix
+   *   Prefix used to identify the fields.
+   */
+  protected static function renderSimpleProductsCustomFields($fields, $sectionTitle, $fieldsIdPrefix) {
+    echo '<div class="options_group show_if_simple show_if_external">';
+    echo '<h3 style="padding-left: 10px;">' . $sectionTitle . '</h3>';
+    foreach ($fields as $field) {
+      woocommerce_wp_text_input([
+        'id' => sprintf('_%s_%s_%s', Plugin::PREFIX, $fieldsIdPrefix, $field['id']),
+        'label' => $field['name'],
+      ]);
```

Comment:
Can we specify the input type as "number"?  We only want numbers as stock values.